src/components/Review.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Review({
+  title,
+  rating,
+  categorie,
+  subcategorie,
+  summary,
+  lessons,
+  index,
+}) {
+  return (
+    <StyledReview key={index}>
```

Comment:
Please don't use the index as a key. The key should always be unique, if you can, add an id to your object and use that as key.

Reply:
I forgot about that one. Will change it.